### PR TITLE
CV

### PR DESCRIPTION
**Pull request by nicopixel**

### CODE REVIEW

File: staff/nicolas-martin/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+        <section id="pre">
+            <header>
+                <h1>Presentación</h1>
+                <p>Lorem fistrum torpedo occaecat a gramenawer velit te va a hasé pupitaa ad.</p>
+            </header>
+
+            <figure>
+                    <img width="100px" src="https://i.pinimg.com/originals/d1/bf/5f/d1bf5fe1b474f25d586ba8150e614773.jpg" alt="Lorem Ipsum">
+                    <figcaption>Yo!</figcaption>
```
Comment (lexbarba) [inline staff/nicolas-martin/cv/index.html]: Cool!

File: staff/nicolas-martin/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        </section>
+
+        <section id="dat">
+            <header>
+                <h1>Datos personales</h1>
+            </header>
+
+            <address>
+                <a href="mailto:ascasdc@gmail.com">jim@rock.com</a><br/>
+                <a href="tel:+311-555-2368">+311-555-2368</a><br/>
```
Comment (lexbarba) [inline staff/nicolas-martin/cv/index.html]: Cool!

File: staff/nicolas-martin/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+        <section id="for">
+            <header>
+                <h1>Formación</h1>
+                <p>Lorem fistrum torpedo occaecat a gramenawer velit te va a hasé pupitaa ad.</p>
+            </header>
+
+            <ul>
+                    <li><h4>Universidad estelar</h4>
+                    From: <time datetime="2008-02-14">01-01-2009</time>
```
Comment (carlosclatg) [inline staff/nicolas-martin/cv/index.html]: Tab first <li>

File: staff/nicolas-martin/cv/index.html (hunk 1 above)
Comment (nicopixel) [reply staff/nicolas-martin/cv/index.html]: Merci!

File: staff/nicolas-martin/cv/index.html (hunk 3 above)
Comment (nicopixel) [reply staff/nicolas-martin/cv/index.html]: sorry but i don't understand 😞

Comment (nicopixel) [reply staff/nicolas-martin/cv/index.html]: I've applied the shortcut in Visual Studio Editor

**Pull request merged**